src/ontology/imports/merged_import.owl

```diff
@@ -13106,7 +13106,7 @@ AnnotationAssertion(rdfs:label <http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(rdfs:label <http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym> "has_broad_synonym")
 AnnotationAssertion(rdfs:seeAlso <http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym> "https://github.com/information-artifact-ontology/ontology-metadata/issues/18")
 
-# Annotation Property: <http://www.geneontology.org/formats/oboInOwl#hasDbXref> (has cross-reference)
+# Annotation Property: <http://www.geneontology.org/formats/oboInOwl#hasDbXref> (database_cross_reference)
```

Comment:
For information: this is of no consequence. This change is because we got conflicting labels for `oboInOwl:hasDbXref`. It will probably take a little while before all conflicting definitions of that property across OBO ontologies are superseded by the now “official” definition in OMO.